config/default/kustomization.yaml

```diff
@@ -34,10 +34,6 @@ patchesStrategicMerge:
 # through a ComponentConfig type
 #- manager_config_patch.yaml
 
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-- manager_webhook_patch.yaml
```

Comment:
If we run kubebuilder again, maybe to add another CRD, will it put this back in place?  Or the manager_webhook_patch.yaml?

Reply:
It probably would, but I think you'd have to run Kubebuilder with the commands to add a web hook. So in that case I think you'd expect to be doing work in this area. Even if this did accidentally get put back in, it wouldn't be doing any damage. It would just be adding the network port to the controller-manager pod.